src/common-model/functions.ts

```diff
@@ -18,16 +18,22 @@
  */
 
 import { DeepReadonly } from 'deep-freeze';
-import { Donor, ClinicalEntity, ClinicalInfo, Treatment } from '../clinical/clinical-entities';
+import _, { isEmpty } from 'lodash';
```

Comment:
we can remove `_` default import and keep using the named exports

Reply:
Updated